.azure/gpu-tests-pytorch.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       matrix:
         # TODO: package parametrization
         'PyTorch - stable':
-          image: "pytorchlightning/pytorch_lightning:base-cuda-py3.9-torch1.12-cuda11.6.1"
```

Comment:
how come we had 11.6.1 here while the docker file had 11.3.1?

Reply:
Because that's the dockerfile default value. The images are constructed from `.github/workflows/release-docker.yml` which you need to update as I did in https://github.com/Lightning-AI/lightning/pull/15347/files#diff-aa38a2216a456b6851247d357e99f30026306296d43bb3c1f843a6614524ef77R22

Reply:
but the name for export is based inputs not defaults